src/26/monitor.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="26" height="26" focusable="false" viewBox="0 0 26 26">
+  <g fill="none">
+    <path d="M0 0h26v26H0z"/>
```

Comment:
This path is superfluous and should be removed. Once that is gone, the `<g>` is no longer needed, either.

Reply:
great catch!